### ISSUE TITLE
Right alt key does not work e.g. right-alt-D

### ISSUE THREAD

**Issue by KBMD** [brave/brave-browser#29450]
## Description 
Today, suddenly, right-alt-D no longer brings up the URL bar. Left-alt-D does. 

## Steps to Reproduce

   1. Open Brave browser.
   2. Press right-alt-D

## Actual result:
Nothing happens

## Expected result:
Open URL bar

## Reproduces how often: 
Today always happens. Didn't happen yesterday or before.

## Brave version (brave://version info)
1.49.120 Chromium: 111.0.5563.64 (Official Build) (64-bit) 
Revision | c710e93d5b63b7095afe8c2c17df34408078439d-refs/branch-heads/5563@{#995}
OS | Windows 11 Version 22H2 (Build 22621.1413)

## Version/Channel Information:
- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 
-
I have not tried this.

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?    NO.
- Does the issue resolve itself when disabling Brave Rewards?    NO
- Is the issue reproducible on the latest version of Chrome?   do not know

## Miscellaneous Information:
It's not the keyboard, as right-alt-D opens the similar bar in Windows Explorer and activates the menu in Microsoft applications. I'm using the US QWERTY keyboard. Right-alt-D doesn't seem to activate any other program, and I haven't installed anything recently.

**Comment by KBMD** [comment]
P.S. thank you to the maintainers. Brave is cool.

**Comment by AbdelAzizMohamedMousa** [comment]
Based on the information provided, it appears that there is an issue with the Brave browser's functionality related to the right-alt-D keyboard shortcut. As left-alt-D works correctly, it suggests that the issue is not with the keyboard itself.

To try and resolve the issue, you can try the following steps:

Check for Brave browser updates: Make sure you are using the latest version of the Brave browser. You can do this by going to the "About Brave" section in the browser settings.

Clear browser cache and cookies: Clearing your browser cache and cookies can often help resolve issues with browser functionality. You can do this by going to the "Clear browsing data" section in the browser settings.

Disable browser extensions: Sometimes browser extensions can cause issues with browser functionality. Try disabling all browser extensions and then see if the issue persists.

Reset browser settings: Resetting your browser settings to default can often help resolve issues. You can do this by going to the "Settings" section in the browser and selecting "Reset settings".

If the issue still persists after trying the above steps, you may want to consider contacting Brave support for further assistance.

**Comment by fallaciousreasoning** [comment]
Hey, thanks for the report @KBMD! If you don't mind, I've got a few questions which will help me work out whats going on so hopefully I can fix it :laughing:.
1. If you create a new profile, does this still happen in the new profile?
2. Does the <kbd>Ctrl</kbd>+<kbd>L</kbd> shortcut still work to focus the location bar (with both control keys)
3. Does it happen in the latest Nightly?
4. Do you have the brave://flags/#brave-commands flag enabled (this shouldn't affect 1.49 but can't hurt to know)? We've been working on some new functionality to let you configure shortcuts, and hopefully I haven't broken something.

**Comment by KBMD** [comment]
I updated Brave and (whether that's the reason or not) it works correctly
now. I had not known about Ctrl+L, but that works too (either control key).

Kevin J. Black

**Comment by fallaciousreasoning** [comment]
Sweet, glad to hear it's working for you now @KBMD!

Hahahaha yeah, I usually use Ctrl+L for it because `L` is for Location in my head :laughing: 

I'm going to close this, but feel free to reopen if the issue comes back (and if you notice anything which triggers it, it'd be super helpful).

**Comment by fallaciousreasoning** [comment]
Okay, that's pretty weird :/ Was there anything you were doing at the time which you think could've been related?

One other thing we could check next time it happens is whether restarting the browser (without updating) fixes the issue, or whether it's just updating. To be honest, I'm a little lost as to what's going on :laughing:

**Comment by KBMD** [comment]
Next time I'll try to remember to check version, restart Brave, etc. Don't
need to keep the issue open until then.


Kevin J. Black

**Comment by fallaciousreasoning** [comment]
Thanks @KBMD - hopefully we can work out what's going on!

**Comment by NemanSyed** [comment]
Found this because I was facing the same problem in Chrome. From what I understood, the right ALT key on US keyboards maps to the AltGr key on non-US keyboards. Chromium may be starting to adhere more forcefully to some accessibility standards thing (as per Gemini's explanation) which results in the right ALT key being interpreted as AltGr even in the context of a US keyboard. There seems to be no way to override that. 👎 

When pressing LEFT-ALT-D it always works. When pressing RIGHT-ALT-D (i.e. AltGr-D) the response is context-dependent. In Chrome (and probably all Chromium browsers), in Google Search, it triggers the "press / to search" overlay instead of the address bar focus. On any other page (like this GitHub page), pressing RIGHT-ALT-D (i.e. AltGr-D) has no function and is ignored.

In other words, it's not a bug, it's a feature. 🙄

Your only option is LEFT-ALT-D unless you want to use the "universal" shortcut of CTRL-L. Both CTRL keys are treated the same regardless of keyboard language hardware.

Sometimes I feel too old to relearn keyboard shortcuts I've used for decades.
